docs/en/integrations/index.mdx

```diff
@@ -169,7 +169,7 @@ We are actively compiling this list of ClickHouse integrations below, so it's no
 |Vector|<img src={require('./images/logos/vector.png').default} class="image" alt="Vector logo" style={{width: '3rem', 'background-color': 'transparent'}}/>|Data ingestion|A lightweight, ultra-fast tool for building observability pipelines with built-in compatibility with ClickHouse.|[Documentation](/docs/en/integrations/vector/)|
 |YepCode||Data integration|YepCode is the integration & automation tool that loves source code.|[Documentation](https://yepcode.io/docs/integrations/clickhouse/)|
 |Zing Data|<img src={require('./images/logos/zing-logo.png').default} class="image" alt="Zing logo" style={{width: '3rem', 'background-color': 'transparent'}}/>|Data visualization|Simple social business intelligence for ClickHouse, made for iOS, Android and the web.|[Documentation](https://docs.getzingdata.com/docs/)|
-|Streamkap|<img src={require('./images/logos/streamkap-logo.png').default} class="image" alt="Streamkap logo" style={{width: '3rem', 'background-color': 'transparent'}}/>|Data ingestion|Streamkap is the modern streaming platform for data teams who want to start syncing data in real-time without the need to hire a team.|[Documentation](https://docs.streamkap.com/docs/clickhouse)|
+|Streamkap|<img src={require('./images/logos/streamkap-logo.png').default} class="image" alt="Streamkap logo" style={{width: '3rem', 'background-color': 'transparent'}}/>|Data ingestion|Setup real-time CDC streaming to ClickHouse with high throughput in minutes.|[Documentation](https://docs.streamkap.com/docs/clickhouse)|
```

Comment:
Should probably define what _CDC_ means in this context. Can you add the string that `CDC` represents?